engine/src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -7,6 +7,7 @@ import akka.actor.ActorSystem
 import better.files._
```

Comment:
Yep no worries. I'm not worrying too much about that right now, I prefer to focus on making sure that the implementation works. I'll rebase when your PR is merged and use your version.